src/vanta_ledger/models/user_models.py

```
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+"""
+User Models for Vanta Ledger
+Defines user data models and database schemas
+"""
+
+from datetime import datetime
+from typing import Optional, List
```

Ruff (0.12.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)